os_sensor:
  ros__parameters:
    sensor_hostname: 'os-122151001683.local'
    udp_dest: ' '
    lidar_mode: ' '
    timestamp_mode: ' '
    udp_profile_lidar: ' '
    metadata: ' '
    lidar_port: 48853
    imu_port: 37028
os_cloud:
  ros__parameters:
    tf_prefix: ' '
    timestamp_mode: ' '
